.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
     rev: v0.971
     hooks:
     -   id: mypy
-        additional_dependencies: [types-PyYAML, types-requests]
+        additional_dependencies: [types-PyYAML, types-requests, types-pkg-resources]
```

Comment:
Is pkg resources used in the plugin?

Reply:
Updated https://github.com/ApeWorX/ape-alchemy/pull/27/commits/27f50f3a58f3bf0f96d06bcb5f85d4f461c7fd96